corehq/apps/dhis2/forms.py

```diff
@@ -13,10 +13,20 @@
 from corehq.apps.style import crispy as hqcrispy
 
 
+LOG_LEVEL_CHOICES = (
+    (logging.CRITICAL, 'Critical'),
+    (logging.ERROR, 'Error'),
+    (logging.WARNING, 'Warning'),
+    (logging.INFO, 'Info'),
+    (logging.DEBUG, 'Debug'),
+)
```

Comment:
Should only include the ones we actually log things at - so at present it would just be `error` and `info`